app/src/main/java/com/onboarding/parkingsystemjava/mvp/view/ParkingView.java

```diff
@@ -0,0 +1,24 @@
+package com.onboarding.parkingsystemjava.mvp.view;
+
+import android.app.Activity;
+import android.widget.Toast;
+
+import com.onboarding.parkingsystemjava.databinding.ActivityMainBinding;
+import com.onboarding.parkingsystemjava.mvp.MainActivityContract;
+
+
+public class ParkingView extends ActivityView implements MainActivityContract.MainActivityView {
+
+    private final ActivityMainBinding binding;
+
+    public ParkingView(Activity activity, ActivityMainBinding binding) {
+        super(activity);
+        this.binding = binding;
+    }
+
+
+    @Override
+    public void showParkingLotsAvailable(int parkingLots) {
+        Toast.makeText(getActivity(), "Se establecieron " + parkingLots + " estacionamientos.", Toast.LENGTH_LONG).show();
```

Comment:
1-. do no hardcode pls, from here you should use the resources
2-. before using the getActivity() you have to validate that it is not null